Livecheckables/gradle.rb

```diff
@@ -0,0 +1,6 @@
+class Gradle
+  livecheck do
+    url "https://github.com/gradle/gradle.git"
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url "https://services.gradle.org/distributions/"
    regex(/href=.*?gradle-v?(\d+(?:\.\d+)+)-all\.(?:[tz])/i)
```

The formula uses an archive file from https://services.gradle.org/distributions/, so it's more appropriate to check that index page.

Though the Git tags are convenient, we try to check as close to the formula's stable archive location as possible. If that's not possible or there's an issue with that approach, then we start checking other sources (e.g., Git tags).